tools/qontract_cli.py

```diff
@@ -0,0 +1,72 @@
+import json
+import click
+
+import utils.gql as gql
+import utils.config as config
+import reconcile.queries as queries
+
+from reconcile.cli import config_file
+
+
+@click.group()
+@config_file
+@click.pass_context
+def root(ctx, configfile):
+    ctx.ensure_object(dict)
+    config.init_from_toml(configfile)
+    gql.init_from_config()
+
+
+@root.group()
+@click.pass_context
+def get(ctx):
+    pass
+
+
+@get.command()
+@click.pass_context
+def settings(ctx):
+    settings = queries.get_app_interface_settings()
+    print(json.dumps(settings))
+
+
+@get.command()
+@click.pass_context
+def aws_accounts(ctx):
+    accounts = queries.get_aws_accounts()
+    print(json.dumps(accounts))
```

Comment:
Why json? Will the output be used as input somewhere?

Reply:
By the way, the `| jq` can be avoided by using `json.dumps(..., indent=2)`. Without colors nonetheless.

Reply:
`-o table|json|yaml|`! :)